src/main/java/core/basesyntax/services/FileCreator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.services;
+
+public interface FileCreator {
+    void put(String report, String filePath);

```

Comment:
put - is not a good name for this method in FileCreator interface.
createFile() is brtter